fjord/feedback/tests/test_views.py

```diff
@@ -102,6 +103,23 @@ def test_valid_sad(self):
         eq_(u'Firefox', feedback.product)
         eq_(u'14.0.1', feedback.version)
 
+    @with_waffle('thankyou', True)
+    def test_thank_you(self):
+        """Thanks_sad template is used when appropriate"""
+        url = reverse('feedback', args=(u'firefox',))
+        r = self.client.post(url, {
+            'happy': 0,
+            'description': u"I want to know why Firefox doesn't make me sandwiches!",
+        }, follow=True)
+
+        feedback = models.Response.objects.latest(field_name='id')
+        eq_(u"I want to know why Firefox doesn't make me sandwiches!", feedback.description)
+        eq_(u'', feedback.url)
+        eq_(False, feedback.happy)
+        eq_(u'en-US', feedback.locale)
+        eq_(r.status_code, 200)
+        self.assertTemplateUsed(r, 'feedback/thanks_sad.html')
```

Comment:
This test is interested in the status code and the template used. Verifying the other things probably isn't important.